content/auth/capabilities.textile

```diff
@@ -514,7 +514,7 @@ It is possible for JWTs to contain authenticated claims for users that can be us
 
 Messages can be annotated with trusted metadata copied from the client's authentication token by Ably servers. Clients are unable to directly publish messages with user claim metadata, and claims contained within the authentication token are signed to prevent tampering. Claims can be scoped to individual channels or to namespaces of "channels":/docs/channels. The most specific user claim will be added to the message as part of the @extras@ object. Note that this does not apply to presence or metadata messages.
 
-An example use case is when using "message interactions":/docs/channels/messages#interactions. You might want to use trusted claims to define 'moderator' users in a chat channel who have the ability to delete any sent messages. When the moderator sends an interaction to mark messages in a channel as deleted, your application should check that user's claims to verify they are a moderator for that channel before actioning their request.
+An example use case is an interactive live streaming sports app. You might want to use trusted claims to define 'moderator' users in a chat channel who have the ability to delete any sent messages. When the moderator sends an interaction to mark messages in a channel as deleted, your application should check that user's claims to verify they are a moderator for that channel before actioning their request.
```

Comment:
This is still referencing use of message interactions with "When the moderator sends an interaction..."

Reply:
Is it worth having links to relevant pages as well?